Chore: Add no-inline-styles hint to default configurations

bolah2009 opened the pull request:
<!--

Read our pull request guide:
https://webhint.io/docs/contributor-guide/getting-started/pull-requests/

For the following items put an "x" between the square brackets
(i.e. [x]) if you completed the associated item.

-->

## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [x] Added/Updated related tests.

## Short description of the change(s)

<!--

If this is a non-trivial change, include information such as what
benefits this change brings as well as possible drawbacks.

If this fixes an existing issue, include the relevant issue number(s).

Thank you for taking the time to open this PR!

-->

This PR 

- [x] Adds `no-inline-styles` hint to default configurations as follow-up PR of https://github.com/webhintio/hint/pull/4161
- [x]  I changed the `severity` from `hint` to `warning` 
- [x]  The category is also changed from 'other' to 'pitfalls'
- [x]  I use loop to go through all the selected elements to report affected element each

antross commented on packages/configuration-development/package.json:
```diff
@@ -17,6 +17,7 @@
     "@hint/hint-meta-charset-utf-8": "^4.0.3",
     "@hint/hint-meta-viewport": "^5.0.3",
     "@hint/hint-no-bom": "^4.2.7",
+    "@hint/hint-no-inline-styles": "^1.0.0",
```
Looks like I missed this on your original PR, but we should change the version number in all the references and in the `package.json` for `hint-no-inline-styles` to `0.0.1`. 

Our release script expects something less than `1.0.0` as part of calculating the initial changelog when a package hasn't been published yet. The script will then bump it to `1.0.0` as part of the release.

```suggestion
    "@hint/hint-no-inline-styles": "^0.0.1",
```

bolah2009 replied:
Thanks, @antross, I just updated it!

bolah2009 commented:
1. I agree that we should change to `warning`
2. I will change the category as well to `pitfalls`
3. I will try this too

Do you want me to do this in a new PR or add them here? @antross

antross commented:
@bolah2009 I'm okay with tackling those changes here if you want since I see them as part of getting this turned on by default. That said, I'm also okay with a separate PR if you'd prefer, but I wouldn't merge this one until those changes are in.

bolah2009 commented:
@antross I have made the changes in this PR.
1. I changed the `severity` from `hint` to `warning` 
2. The category is also changed from 'other' to 'pitfalls'
3. I use loop to go through all the selected elements to report affected element each

antross reviewed (approved):
Looks good to me! Thanks again @bolah2009 for the updates 🎉

@sarvaje do you want to take a look at this before I merge?

FYI I built the browser extension again locally with these updates and confirmed this is working as expected.

sarvaje reviewed (approved):
LGTM
